<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpConnForm</class>
 <widget class="QWidget" name="TcpConnForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lblIpTitle">
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leIpAddress"/>
   </item>
   <item>
    <widget class="QLabel" name="lblPortTitle">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sbPort"/>
   </item>
   <item>
    <widget class="QLabel" name="lblServerIdTitle">
     <property name="text">
      <string>Server ID:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sbServerId"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>Disconnected</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
